pkg/kptfile/pkgfile.go

```diff
@@ -270,10 +252,9 @@ func shouldRemoveValue(updatedDef, localDef, originalDef *yaml.MapNode, key stri
 type Dependency struct {
 	Name            string `yaml:"name,omitempty"`
 	Upstream        `yaml:",inline,omitempty"`
-	EnsureNotExists bool       `yaml:"ensureNotExists,omitempty"`
-	Strategy        string     `yaml:"updateStrategy,omitempty"`
-	Functions       []Function `yaml:"functions,omitempty"`
-	AutoSet         bool       `yaml:"autoSet,omitempty"`
+	EnsureNotExists bool   `yaml:"ensureNotExists,omitempty"`
+	Strategy        string `yaml:"updateStrategy,omitempty"`
```

Comment:
I see it now, so one can specify in the dependencies what functions to invoke (set is one trigger point).

Reply:
Per comment above, we need to uplevel this instead of having a separate mechanism for specifying functions in the Kptfile.